File: scala-play-server/app/model/ResponseTimeFilterProperties.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ResponseTimeFilterProperties.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class ResponseTimeFilterProperties(
  travelTime: Option[Int],
  distance: Option[Int],
  distanceBreakdown: Option[List[ResponseDistanceBreakdownItem]],
  fares: Option[ResponseFares],
  route: Option[ResponseRoute]
)

object ResponseTimeFilterProperties {
  implicit lazy val responseTimeFilterPropertiesJsonFormat: Format[ResponseTimeFilterProperties] = Json.format[ResponseTimeFilterProperties]
}

File: fsharp-giraffe-server/OpenAPI/src/model/ResponseTimeFilterProperties.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.ResponseDistanceBreakdownItem
open OpenAPI.Model.ResponseFares
open OpenAPI.Model.ResponseRoute

module ResponseTimeFilterProperties = 

  //#region ResponseTimeFilterProperties


  type ResponseTimeFilterProperties = {
    TravelTime : int;
    Distance : int;
    DistanceBreakdown : ResponseDistanceBreakdownItem[];
    Fares : ResponseFares;
    Route : ResponseRoute;
  }
  //#endregion
  